parquet-arrow/src/main/java/org/apache/parquet/arrow/schema/SchemaConverter.java

```diff
@@ -245,7 +246,14 @@ public TypeMapping visit(Date type) {
 
       @Override
       public TypeMapping visit(Time type) {
-        return primitive(INT32, TIME_MILLIS);
+        int bitWidth = type.getBitWidth();
+        org.apache.arrow.vector.types.TimeUnit timeUnit = type.getUnit();
```

Comment:
This should be imported at the top to have a much shorter code line.

Reply:
@xhochy Thanks for your review. I will fix it.